### PR TITLE
isdefinedby splited to literal and iri based on isdefinedby usage

### PR DESCRIPTION
**Pull request by bernardyn**
Decided to find all usage of "isDefinedBy" phrase. Thanks of that I used a specific groups of results to change to IRI or Literal. @mereolog please have a look at the regex searching phrases here: https://docs.google.com/spreadsheets/d/1ViTgwk3dKOEoONQbMYaj5rvj8gah9A2_4yNrjfmbipY/edit?usp=sharing

### CODE REVIEW

File: ontohgis.ttl [hunk 1]
```diff
@@ -6537,7 +6537,7 @@ ontohgis:administrative_type_1 rdf:type owl:Class ;
...
-                               rdfs:isDefinedBy ontohgis:administrative_system_4 ;
+                               rdfs:isDefinedByIRI ontohgis:administrative_system_4 ;
```
Comment (przemekgradzki) [inline ontohgis.ttl:6540]:
```suggestion
                               ontohgis:isDefinedByIRI ontohgis:administrative_system_4 ;
```

The same applies to all instances of _rdfs:isDefinedByIRI_ and _rdfs:isDefinedByLiteral_, i.e. perform the following substitutions:

_rdfs:isDefinedByIRI_ -> _**ontohgis**:isDefinedByIRI_
_rdfs:isDefinedByLiteral_ -> _**ontohgis**:isDefinedByLiteral_

with the following command:
```bash
sed -i -e 's/\(\s\)\S\+:isDefinedByIRI/\1ontohgis:isDefinedByIRI/gi' \
       -e 's/\(\s\)\S\+:isDefinedByLiteral/\1ontohgis:isDefinedByLiteral/gi' ontohgis.ttl
```

File: ontohgis.ttl [hunk 2]
```diff
@@ -10246,7 +10246,7 @@ ontohgis:object_23 rdf:type owl:Class ;
                                          rdf:type owl:Class
                                        ] ;
                    rdfs:subClassOf ontohgis:object_1 ;
-                   rdfs:isDefinedBy "Budynek jest to przedmiot trwający w czasie, który został wytworzony lub wykorzystany w celu odgraniczenia pewnej przestrzeni (“wewnątrz\") od innej przestrzeni (“zewnątrz”)."@pl ;
+                   rdfs:isdefinedbyLiteral "Budynek jest to przedmiot trwający w czasie, który został wytworzony lub wykorzystany w celu odgraniczenia pewnej przestrzeni (“wewnątrz\") od innej przestrzeni (“zewnątrz”)."@pl ;
```
Comment (przemekgradzki) [inline ontohgis.ttl:10249]:
```suggestion
                   rdfs:isDefinedByLiteral "Budynek jest to przedmiot trwający w czasie, który został wytworzony lub wykorzystany w celu odgraniczenia pewnej przestrzeni (“wewnątrz\") od innej przestrzeni (“zewnątrz”)."@pl ;
```

names are case-sensitive. The same applies to all instances of isdefinedbyIRI and isdefinedbyLiteral.

Use the same command as in the previous comment, i.e.:

```bash
sed -i -e 's/\(\s\)\S\+:isDefinedByIRI/\1ontohgis:isDefinedByIRI/gi' \
       -e 's/\(\s\)\S\+:isDefinedByLiteral/\1ontohgis:isDefinedByLiteral/gi' ontohgis.ttl
```

**Review by przemekgradzki — CHANGES_REQUESTED** [review]

File: ontohgis.ttl (hunk 2 above)
Comment (bernardyn) [reply ontohgis.ttl:10249]: Agree. Fixed.

File: ontohgis.ttl (hunk 1 above)
Comment (bernardyn) [reply ontohgis.ttl:6540]: @mereolog can you have a look?

Comment (mereolog) [reply ontohgis.ttl:6540]: > @mereolog can you have a look?

I did - see: https://github.com/ontohgis/ontology/pull/14#pullrequestreview-1418508728.

Comment (bernardyn) [reply ontohgis.ttl:6540]: Thanks, @mereolog  but I do not understand... Simply, I need a decision do we can merge my changes or not.

Comment (mereolog) [reply ontohgis.ttl:6540]: > The same applies to all instances of _rdfs:isDefinedByIRI_ and _rdfs:isDefinedByLiteral_, i.e. perform the following substitutions:
> 
> _rdfs:isDefinedByIRI_ -> _**ontohgis**:isDefinedByIRI_ _rdfs:isDefinedByLiteral_ -> _**ontohgis**:isDefinedByLiteral_
> 
> with the following command:
> 
> ```shell
> sed -i -e 's/\(\s\)\S\+:isDefinedByIRI/\1ontohgis:isDefinedByIRI/gi' \
>        -e 's/\(\s\)\S\+:isDefinedByLiteral/\1ontohgis:isDefinedByLiteral/gi' ontohgis.ttl
> ```

I agree with these comments.

Comment (bernardyn) [reply ontohgis.ttl:6540]: @mereolog great, thanks. 
@przemekgradzki Please have a look once again at performed changes I already pushed.

**Review by przemekgradzki — APPROVED** [review]

**Review by przemekgradzki — APPROVED** [review]

**Review by mereolog — APPROVED** [review]

**Pull request merged**